docs/admin-guide/load-config.md

```diff
@@ -1,125 +1,92 @@
-# The load config
+# The Load Config File
+
 
 Scout have the possibility to store loads of information about a case and the samples that are included. It is cumbersome to specify to many parameters on the command line so there is an option to give this information in a yaml formated config file.
 Here we can give scout some meta information about the analysis, how it was performed, information about family, samples etc.
 
-The basic structure of a load config looks like:
-
-
-```yaml
```

Comment:
I don't think you should remove this part, because this is the template that users can copy and modify to load  their own data, It's already formatted as a yaml so it's very useful to have. Moreover it shows how the yaml syntax works.